redis_json/src/commands.rs

```diff
@@ -354,10 +354,13 @@ pub fn json_mset<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>)
                 redis_key.set_value(Vec::new(), value)?
             };
             if updated {
-                redis_key.apply_changes(ctx, "json.mset")?
+                redis_key.notify_keyspace_event(ctx, "json.mset")?
```

Comment:
I think the separation you are doing here is right, manager should probably not handle the replication. But I believe we should make this separation all over the code, WDYT?
In addition, maybe we should find a more general name to it? Because on HDT it also need to apply the changes on the key and fire the effects.

Reply:
how it's managed in redis? there are calling to replication and notify in the commands? or it's something that happened in the behind the scenes?

Reply:
Notify is per command.
Replication is a totally different mechanism.